src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/Workloads.drl

```diff
@@ -152,3 +152,35 @@ rule "Workloads_Microsoft_SQL_Server_On_Windows"
                addWorkload("Microsoft SQL Server")
            }
 end
+
+rule "Workloads_System_Services_Present"
```

Comment:
I know the rules and in the `Workload.drl` but we could use a slightly different name for them in order to help understanding they are not adding any workload.
```suggestion
rule "SsaEnabled_System_Services_Present"
```